outcomes/firefox_ios/default_browser.toml

```diff
@@ -0,0 +1,37 @@
+friendly_name = "Default browser"
+description = "Metrics that describe whether Firefox for iOS is a user's default browser."
+
+[metrics.default_browser_action]
```

Comment:
I copied this metric from the fenix outcome: https://github.com/mozilla/jetstream-config/blob/08b4365aca5d57352e628a03111beb03122ead88/outcomes/fenix/default-browser.toml#L25

Reply:
I don't believe we're able to track this (default_browser_action) for iOS. We have to use the proxy metric that measures whether they opened an external link through Firefox, which is our best indication of whether we are the default browser or not.

Reply:
I've removed `default_browser_action` for now. We can always add more metrics later and update how we compute the metrics if we need to.